instrumentation/kafka-clients/src/test/java/brave/kafka/ITTracingKafka.java

```diff
@@ -0,0 +1,89 @@
+package brave.kafka;
+
+import brave.Tracing;
+import brave.sampler.Sampler;
+import com.github.charithe.kafka.EphemeralKafkaBroker;
+import com.github.charithe.kafka.KafkaJunitRule;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.concurrent.Future;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.TopicPartition;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import zipkin.Span;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+public class ITTracingKafka {
+
+  String TEST_TOPIC = "myTopic";
+  String TEST_KEY = "foo";
+  String TEST_VALUE = "bar";
+
+  Tracing consumerTracing;
+  Tracing producerTracing;
+
+  LinkedList<Span> consumerSpans = new LinkedList<>();
+  LinkedList<Span> producerSpans = new LinkedList<>();
+
+  @Rule
+  public KafkaJunitRule kafkaRule = new KafkaJunitRule(EphemeralKafkaBroker.create());
+
+  @Before
+  public void setUp() throws Exception {
+    consumerTracing = Tracing.newBuilder()
```

Comment:
you can initialize these as fields.. no need to do this at setup. Also you can initialize them as KafkaTracing.

Add the following after to ensure thread local stuff doesn't stain other tests
```java
  @After public void close() throws Exception {
    Tracing current = Tracing.current();
    if (current != null) current.close();
  }
```